Gemfile

```diff
@@ -64,3 +64,4 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'date_validator'
```

Comment:
Here I add date_validator because in the booking model we use its syntax to validate end_date after or equal to start_date